content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -94,7 +94,7 @@ kube-apiserver -h | grep enable-admission-plugins
 In the current version, the default ones are:
 
 ```shell
-NamespaceLifecycle, LimitRanger, ServiceAccount, TaintNodesByCondition, Priority, DefaultTolerationSeconds, DefaultStorageClass, StorageObjectInUseProtection, PersistentVolumeClaimResize, RuntimeClass, CertificateApproval, CertificateSigning, CertificateSubjectRestriction, DefaultIngressClass, MutatingAdmissionWebhook, ValidatingAdmissionWebhook, ResourceQuota
+AlwaysAdmit, AlwaysDeny, AlwaysPullImages, CertificateApproval, CertificateSigning, CertificateSubjectRestriction, DefaultIngressClass, DefaultStorageClass, DefaultTolerationSeconds, DenyEscalatingExec, DenyExecOnPrivileged, DenyServiceExternalIPs, EventRateLimit, ExtendedResourceToleration, ImagePolicyWebhook, LimitPodHardAntiAffinityTopology, LimitRanger, MutatingAdmissionWebhook, NamespaceAutoProvision, NamespaceExists, NamespaceLifecycle, NodeRestriction, OwnerReferencesPermissionEnforcement, PersistentVolumeClaimResize, PersistentVolumeLabel, PodNodeSelector, PodSecurityPolicy, PodTolerationRestriction, Priority, ResourceQuota, RuntimeClass, SecurityContextDeny, ServiceAccount, StorageObjectInUseProtection, TaintNodesByCondition, ValidatingAdmissionWebhook
```

Comment:
This doesn't look right to me. Isn't this just all of them? I think the default list is defined here: https://github.com/kubernetes/kubernetes/blob/1119a505aca14467accedf850daf30aa9c532ef2/pkg/kubeapiserver/options/plugins.go#L143-L161

Reply:
Good catch.  I cut-and-pasted the wrong list.  That said, is it really worthwhile to list "in the current version" in docs - that doesn't help much.  What version?  Maybe we should just nix this block?

Reply:
I'll update the list, but still think we should remove it totally.

Reply:
Yeah, I think just giving the command should be sufficient, or maybe even linking to the code. I could go either way.

Reply:
If I'm remembering this correctly, the order of this list matters. Is that still the case? I just noticed that the list is now in alphabetic order.

Reply:
When using --enable-admission-plugins/--disable-admission-plugins, the order is unimportant. That just toggles them on/off, but the order is predetermined.